### DISCUSSION TITLE
Disable automatically creating daily note

### DISCUSSION THREAD

**Discussion by richardrl** [athensresearch/athens#1330] [Feature Requests]
There should be an option to disable the automatically generated daily note.

**Comment by nishu3210** [comment]
I agree because everyone wants to manage things differently.

**Comment by nishu3210** [comment]
I would like to work on this, but would need some help understanding the current code structure and how I can make change.

**Comment by jefftangx** [comment]
@richardrl @nishu3210 do you not use Daily Notes?

**Comment by lswank** [comment]
Aren’t daily notes cleaned up if they are empty after the day? I thought this was the case. Maybe I’m observing it incorrectly.

**Comment by jefftangx** [comment]
Don't think they are cleaned up by default, thought I think that makes sense!

**Comment by richardrl** [comment]
> @richardrl @nishu3210 do you not use Daily Notes?

Not at all.

**Comment by snird** [comment]
The fact that they aren't cleaned up creates a massive mess in the graph, and there is no easy way to dispose of all empty notes.

**Comment by hoopahmadness** [comment]
I actually don't mind having the daily note created automatically (it makes sense) but perhaps a check that runs daily and on startup to remove the previous notes that are still empty. There doesn't even need to be a setting for this right? I agree that cleaning up empty notes is really annoying.

**Comment by ithakaa** [comment]
Scrolling back a few days to find a note is very non productive if you've been I'll and want to find your last note

Something needs to be done about empty notes
